docs/tutorial/workers.md

```diff
@@ -48,12 +56,10 @@ graph TD
 
     worker --> |polls| work_pool
     worker --> |creates| flow_run_infra
+    worker --> |creates| flow_run_infra_2
 ```
 
-!!! note "Security Note"
-    Prefect provides execution through its hybrid model, which allows you to deploy workflows that run in the environments best suited to their execution while allowing you to keep your code and data completely private. 
-    There is no ingress required. 
-    For more information [read more about our hybrid model](https://www.prefect.io/security/overview/#overview).
+<sup>Notice above that the worker is in charge of provisioning the _flow run infra_. In context of this tutorial, that flow run infra is an ephemeral docker container to host each flow run. Different [worker types](/concepts/work-pools/#worker-types) can create different types of flow run infrastructure.</sup>
```

Comment:
This is what i say to people when I walk them through the diagram and it usually results in an aha moment from them.